src/interfaces/gtk3/ec_gtk3.c

```diff
@@ -1748,7 +1748,11 @@ gboolean gtkui_context_menu(GtkWidget *widget, GdkEventButton *event, gpointer d
    (void) widget;
 
     if(event->button == 3) {
+#if GTK_CHECK_VERSION(3,22,0)
```

Comment:
is this true for say gtk 3.22.26?

Reply:
> is this true for say gtk 3.22.26?

yes: https://developer.gnome.org/gtk3/stable/gtk3-Feature-Test-Macros.html#GTK-CHECK-VERSION:CAPS